app/models/consistency_checklist.rb

```diff
@@ -23,6 +31,16 @@ class ConsistencyChecklist < ApplicationRecord
     enum(check => { to_be_determined: 0, yes: 1, no: 2 }, _prefix: check)
   end
 
+  REQUEST_TYPES.each do |check, request_type|
```

Comment:
Is all this meta-coding unreadable?!

Reply:
In truth it's readable within the file... it's not great but you can work it out - to be fair the array/hash of items creating methods, REQUEST_TYPES in this case, it's quite common.

What I didn't work out is how these records interacts with the rest of the application. I was about to put in "raise 'error'" and thought, "I shouldn't have to do that to know what's happening. I rely on "grepping" the code but it's more difficult when you don't have a unique word to search for. If I am not alone in the confusion maybe a comment??

Reply:
Yes, that sounds like a good idea! Maybe just comments listing the public methods that are defined by each loop?

eg above line 35 `# defined #defaut_description_matches_documents_to_no?, #defaut_documents_consistent_to_no?, #defaut_site_map_correct_to_no?`

That way anyone searching the application to find where the method is defined will end up in the right place!

Reply:
It helps having the full name as I can see where it's used but I can't tell if I'm being thick or not.